From 9c1152332d4132acdd84aa014dbd9408eac5ce02 Mon Sep 17 00:00:00 2001
From: Binh Nguyen <binh.nguyen.uw@renesas.com>
Date: Sat, 28 Sep 2019 13:46:01 +0700
Subject: [PATCH] ARM64: DTS: r8a774c0: change gpu power domain to PD_3DG_B

This change power domain of RZ/G2E GPU to support ES1.1

Signed-off-by: Binh Nguyen <binh.nguyen.uw@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774c0.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a774c0.dtsi b/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
index 92365b1..4f8081a 100644
--- a/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
@@ -1764,7 +1764,7 @@
 			interrupts = <GIC_SPI 119 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&cpg CPG_MOD 112>;
 			operating-points-v2 = <&gsx_opp_table>;
-			power-domains = <&sysc R8A774C0_PD_3DG_A>;
+			power-domains = <&sysc R8A774C0_PD_3DG_B>;
 			resets = <&cpg 112>;
 		};
 
-- 
2.7.4

